tox_packet/src/dht/cookie.rs

```diff
@@ -278,7 +283,7 @@ mod tests {
             payload: vec![43; 88]
         };
         let cookie_2 = EncryptedCookie {
-            nonce: secretbox::gen_nonce(),
+            nonce: [43; xsalsa20poly1305::NONCE_SIZE],
```

Comment:
43?)))

Reply:
Yes, because cookie_2 is different from cookie_1 :)